be/src/vec/sink/vtablet_sink.cpp

```diff
@@ -1049,6 +1175,18 @@ Status VOlapTableSink::prepare(RuntimeState* state) {
         return Status::InternalError("unknown destination tuple descriptor");
     }
 
+    for (size_t idx = 0; idx < _output_tuple_desc->slots().size(); idx++) {
+        if (_output_tuple_desc->slots()[idx]->is_auto_increment()) {
+            _auto_inc_col_idx = idx;
+            _auto_inc_id_buffer = GlobalAutoIncBuffers::GetInstance()->get_auto_inc_buffer(
```

Comment:
why here need a global Auto Buffer？

Reply:
There may be multiple `VOlapTableSink`s on one BE writing to the same table concurrently. Sharing a global buffer between these `VOlapTableSink`s can reduce the number of RPCs